website/versioned_docs/version-1.1/appd-spec.md

```diff
@@ -0,0 +1,16 @@
+---
+sidebar_label: App Directory Specification
+title: App Directory Specification 1.0
+original_id: appd-spec
+---
+## API
+
+View the [full specification](/1.0/appd-spec) in [OpenAPI v3.0](https://www.openapis.org/) format (generated with [ReDoc](https://rebilly.github.io/ReDoc/)).
```

Comment:
```suggestion
View the full specification in [OpenAPI v3.0](https://www.openapis.org/) format (generated with [ReDoc](https://rebilly.github.io/ReDoc/)).
```

The "full specification" link is broken, which is contributing to this PR's build errors. However, if you fix it, all it does is link you to... the same page? I'd suggest just removing the link entirely. I have tested this change.

Reply:
also fixed